app/services/risk_assessment/high_risk_assessment.rb

```diff
@@ -6,17 +6,14 @@ class HighRiskAssessment
 
     def initialize(claim)
       @claim = claim
-
-      @summary = Nsm::CheckAnswers::CostSummaryCard.new(claim)
     end
 
     def assess
       high_cost? || counsel_assigned? || enhanced_rates_claimed? || extradition? || rep_order_withdrawn?
     end
 
     def high_cost?
-      profit_cost_summary = @summary.table_fields(formatted: false).find { _1[:name] == 'profit_costs' }
-      profit_cost_summary[:gross_cost] >= HIGH_RISK_THRESHOLD
+      @claim.totals[:cost_summary][:profit_costs][:claimed_total_inc_vat] >= HIGH_RISK_THRESHOLD
```

Comment:
Since this threshold is shared between the caseworker and provider apps - would it make sense to encorporate it into the common gem in the future?

Reply:
Thoroughly agree! I've pitched doing exactly that in my "Move the biz logic around" ADR: https://dsdmoj.atlassian.net/wiki/spaces/CRM457/pages/5246943990/Draft+ADR+16+Business+logic+placement#%E2%9D%8C-D.-The-provider-app-is-currently-responsible-for-flagging-claims-as-high-risk